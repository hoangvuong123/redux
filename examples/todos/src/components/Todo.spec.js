import React from 'react'
import renderer from 'react-test-renderer';
import Todo from './Todo'

describe('Todo component', () => {
  it('render Todo component', () => {
    let onClick = () => "onClick Function"

    const todo = renderer.create(
      <Todo onClick={onClick} completed={true} text="todo 1"/>
    ).toJSON()
    expect(todo).toMatchSnapshot();
  })
})
