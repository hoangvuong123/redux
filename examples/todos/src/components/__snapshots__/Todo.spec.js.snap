// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component render Todo component 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "line-through",
    }
  }
>
  todo 1
</li>
`;
